deploy/charts/rook-ceph/values.yaml

```diff
@@ -7,7 +7,7 @@ image:
   repository: docker.io/rook/ceph
   # -- Image tag
   # @default -- `master`
-  tag: v1.15.3
+  tag: v1.15.4
```

Comment:
Would it make sense to use the `appVesrsion` as the tag? Or would that break a CI workflow?